<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camcalibration</class>
 <widget class="QDialog" name="camcalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2-Cam Calibration</string>
  </property>
  <widget class="glwidget2d" name="glwidget1" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="glwidget2d" name="glwidget2" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savePoints">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>357</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Points</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>691</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin Calibration</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxNumWidth">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxNumHeight">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>335</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>339</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Chess Board</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>334</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Points Saved : </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumPoints">
   <property name="geometry">
    <rect>
     <x>633</x>
     <y>335</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getNew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>357</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Get New Points</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearPoints">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>357</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Points</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="useAdaptive">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Adaptive Threshold</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="useFast">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Fast Check</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="useEqualization">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Equalization</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="framesToPause">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>55</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>304</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Frames to pause : </string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glwidget2d</class>
   <extends>QWidget</extends>
   <header>glwidget2d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
